conformance/image.go

```diff
@@ -14,6 +14,9 @@ type manifest struct {
 	// SchemaVersion is the image manifest schema that this image follows
 	SchemaVersion int `json:"schemaVersion"`
 
+	// MediaType specifies the type of this document data structure e.g. `application/vnd.oci.image.manifest.v1+json`
+	MediaType string `json:"mediaType,omitempty"`
```

Comment:
kk.. trying to remember if there was a good reason for us to have our own copy of these types vs importing them from the aligning image spec release

Reply:
I was asking the same thing recently. We are injecting a custom field in the descriptor where we can't just extend the struct in the test (since we would need to modify the manifests that include the descriptor). If we modified the tests to only extend the manifests with custom fields, this could become an import.